models/intermediate/int_ad_reporting__keyword_report.sql

```diff
@@ -1,7 +1,7 @@
 {% if var('twitter_ads__using_keywords', True) %}
-    {% set include_list = ['apple_search_ads', 'google_ads', 'microsoft_ads', 'pinterest_ads', 'twitter_ads'] %}
+    {% set include_list = ['apple_search_ads', 'google_ads', 'microsoft_ads', 'pinterest_ads', 'amazon_ads', 'twitter_ads'] %}
 {% else %}
-    {% set include_list = ['apple_search_ads', 'google_ads', 'microsoft_ads', 'pinterest_ads'] %}
+    {% set include_list = ['apple_search_ads', 'google_ads', 'microsoft_ads', 'pinterest_ads', 'amazon_ads'] %}
```

Comment:
Same particular request to have these be in alphabetical order.

Reply:
Done